### DISCUSSION TITLE
HVTEC

### DISCUSSION THREAD

**Discussion by hluker** [weather-gov/api#366] [Q&A]
Any chance we can have the HVTEC string added to the individual alert features, as we do with the PVTEC right now?

```
                    "VTEC": [
                        "/O.EXT.KDMX.FL.W.0001.000000T0000Z-200312T0000Z/"
                    ],
```

It would be nice to have the individual pieces parsed out (esp. NWSLI), but having at least the raw string would be a great place to start at least.

**Comment by StephenClouse** [comment]
Unfortunately, no. The HVTEC is not provided in the CAP feed from HazCollect, which is our source for the alerts data.
